ch_tools/monrun_checks/dns.py

```diff
@@ -29,30 +29,35 @@ def __init__(self, fqdn: str, private: bool, strict: bool):
         self.strict = strict
 
 
-@click.command("ext_ip_dns")
+@click.command("dns")
 @click.option("--cluster", "cluster", is_flag=True, help="Check cluster DNS records")
 @click.option("--private", "private", is_flag=True, help="Check private DNS records")
+@click.option("--ipv4", "ipv4", is_flag=True, help="Check A DNS records")
```

Comment:
Enabled by default in `config.py`.

Reply:
Why not with `default=True` here?

Reply:
Just to not duplicate defaults from `config.py`.

After #122, if an option is controlled through both config and command-line argument it's possible to specify its default value in only one place - in `config.py`.